autode/opt/coordinates/primitives.py

```diff
@@ -298,12 +299,12 @@ class PrimitiveBondAngle(Primitive):
     arccosine of the normalised dot product
     """
 
-    def __init__(self, o: int, m: int, n: int):
+    def __init__(self, m: int, o: int, n: int):
```

Comment:
@t-young31 Yes, I had to change the tests as well. I felt that it would be more consistent with how mol.angle(i, j, k) is defined (and also consistent with dihedrals order).

Reply:
yeah, I agree. I'm sure there was a reason I wrote it like this initially but it escapes me now. thanks!